checker/bin/mapreduce-checker.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+LAUNCHER=
+# If debugging is enabled propagate that through to sub-shells
+if [[ "$-" == *x* ]]; then
+  LAUNCHER="bash -x"
+fi
+BIN=$(cd "$( dirname "$( readlink "$0" || echo "$0" )" )"; pwd)
+
+USAGE="Usage: alluxio-checker.sh mapreduce [INPUT_SPLITS]
```

Comment:
Our MapReduce job hopes to check more mapper nodes, but the number of mapper nodes is influenced by input splits and seems to be only influenced by input splits. 
I am not able to set the exact mapper number.

The input splits here will define how many input txt files (each file just have a number 1)  I create in the MapReduce input folder.